harness/determined/cli/task.py

```diff
@@ -110,6 +112,122 @@ def logs(args: Namespace) -> None:
         )
 
 
+@authentication.required
+def create(args: Namespace) -> None:
+    config = ntsc.parse_config(args.config_file, None, args.config, [])
+    config_text = util.yaml_safe_dump(config)
+    context_directory = None
+    if args.context is not None:
+        context_directory = context.read_v1_context(args.context, args.include)
+
+    sess = cli.setup_session(args)
+    req = bindings.v1CreateGenericTaskRequest(
+        config=config_text,
+        contextDirectory=context_directory,
+        projectId=args.project_id,
+        parentId=args.parent,
+        inheritContext=args.inherit_context
+    )
+    task_resp = bindings.post_CreateGenericTask(sess, body=req)
+    print(f"created task {task_resp.taskId}")
+
+    if task_resp.warnings:
+        cli.print_warnings(task_resp.warnings)
+
+    if args.follow:
+        try:
+            logs = api.task_logs(sess, task_resp.taskId, follow=True)
+            api.pprint_logs(logs)
+        finally:
+            print(
+                colored(
+                    "Task log stream ended. To reopen log stream, run: "
+                    "det task logs -f {}".format(task_resp.taskId),
+                    "green",
+                )
+            )
+
+
+@authentication.required
+def config(args: Namespace) -> None:
+    sess = cli.setup_session(args)
+    config_resp = bindings.get_GetGenericTaskConfig(sess, taskId=args.task_id)
+    if args.json:
+        render.print_json(config_resp.config)
+    else:
+        yaml_dict = json.loads(config_resp.config)
+        print(util.yaml_safe_dump(yaml_dict, default_flow_style=False))
+
+
+@authentication.required
+def kill(args: Namespace) -> None:
+    sess = cli.setup_session(args)
+    bindings.post_KillGenericTask(sess, taskId=args.task_id, killFromRoot=args.root)
```

Comment:
I tried `det task kill` and it throws an error:

`TypeError: post_KillGenericTask() got an unexpected keyword argument 'killFromRoot'`

I think the protos needs a `body` option:
```
diff --git a/proto/src/determined/api/v1/api.proto b/proto/src/determined/api/v1/api.proto
index 37f29b15d..38cb4d446 100644
--- a/proto/src/determined/api/v1/api.proto
+++ b/proto/src/determined/api/v1/api.proto
@@ -2411,9 +2411,10 @@ service Determined {
       returns (KillGenericTaskResponse) {
     option (google.api.http) = {
       post: "/api/v1/tasks/{task_id}/kill"
+      body: "*"
     };
     option (grpc.gateway.protoc_gen_swagger.options.openapiv2_operation) = {
       tags: "Internal"
     };
   }
```

and then:

```suggestion
    req = bindings.v1KillGenericTaskRequest(taskId=args.task_id, killFromRoot=args.root)
    bindings.post_KillGenericTask(sess, taskId=args.task_id, body=req)
```